dev-support/docker-images/submarine/Dockerfile

```diff
@@ -41,7 +41,7 @@ ADD ./tmp/mysql-connector-java-5.1.39.jar "/opt/submarine-current/lib/"
 # Create submarine Bucket
 WORKDIR /usr/local/bin
 RUN wget https://dl.min.io/client/mc/release/linux-amd64/mc && chmod +x mc
-RUN wget https://dl.k8s.io/release/v1.15.11/bin/linux/amd64/kubectl && chmod +x kubectl
+# RUN wget https://dl.k8s.io/release/v1.15.11/bin/linux/amd64/kubectl && chmod +x kubectl
```

Comment:
Can you remove this line instead of comment it?

Reply:
ok